contrib/babelfishpg_common/sql/geography.sql

```diff
@@ -152,8 +152,16 @@ CREATE OR REPLACE FUNCTION sys.GEOGRAPHY(bytea)
 
 CREATE OR REPLACE FUNCTION sys.bytea(sys.GEOGRAPHY)
 	RETURNS bytea
-	AS '$libdir/postgis-3','LWGEOM_to_bytea'
-	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+	AS $$
+	DECLARE
+        byte bytea;
+	BEGIN
+		byte := (SELECT sys.bytea_helper($1));
+		byte := substring(byte from 6);
+		byte := substring(byte from 1 for 4) || E'\\x010c' || substring(byte from 5);
```

Comment:
what's this value? maybe worth leaving a comment describing what's going on here

Reply:
Done